src/locale-en.js

```diff
@@ -0,0 +1,37 @@
+/**
+* Copyright 2012-2017, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+module.exports = {
+    moduleType: 'locale',
+    name: 'en',
+    dictionary: {
+        'Click to enter Colorscale title': 'Click to enter Colourscale title'
+    },
+    format: {
+        days: ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday'],
+        shortDays: ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'],
+        months: [
+            'January', 'February', 'March', 'April', 'May', 'June',
+            'July', 'August', 'September', 'October', 'November', 'December'
+        ],
+        shortMonths: [
+            'Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun',
+            'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'
+        ],
+        periods: ['AM', 'PM'],
```

Comment:
I'm curious here. Do locales w/o `periods` set fallback to using a 24-hour clock?

Reply:
`periods` is only relevant if you use `%p` (and `%I` for 12-hour hours) in a custom format, our standard formatting always uses a 24-hour clock. Locales that don't define `periods` will inherit `['AM', 'PM']` from `en`. Seems like most places don't care about this, and some that do are still OK with AM/PM (since it's latin anyway).